src/Components/Server/ref/Microsoft.AspNetCore.Components.Server.csproj

```diff
@@ -7,11 +7,12 @@
     <Compile Include="Microsoft.AspNetCore.Components.Server.netcoreapp3.0.cs" />
     <Reference Include="Microsoft.AspNetCore.Components.Web"  />
     <Reference Include="Microsoft.AspNetCore.DataProtection"  />
-    <Reference Include="Microsoft.Extensions.Logging"  />
     <Reference Include="Microsoft.AspNetCore.SignalR"  />
     <Reference Include="Microsoft.AspNetCore.StaticFiles"  />
     <Reference Include="Microsoft.Extensions.Caching.Memory"  />
     <Reference Include="Microsoft.Extensions.FileProviders.Composite"  />
     <Reference Include="Microsoft.Extensions.FileProviders.Embedded"  />
+    <Reference Include="Microsoft.Extensions.Logging"  />
+    <Reference Include="Microsoft.Extensions.ValueStopwatch.Sources"  />
```

Comment:
Do we need this in the ref assembly project?

Reply:
Hmm, I guess I'm missing `PrivateAssets="All"`. The codegen complained about it.

Reply:
that's bizzard

Reply:
.Sources automagically get PrivateAssets set

Reply:
Maybe that's what https://github.com/aspnet/AspNetCore/pull/12638#discussion_r307980162 was about. It doesn't work unless I explicitly set it.